src/NUnitFramework/tests/TestContextTests.cs

```diff
@@ -221,10 +222,19 @@ public void SetUpCanAccessTestId()
         #region Properties
 
         [Test]
-        [Property("Answer", 42)]
+        [Property("Answer", "42")]
         public void TestCanAccessItsOwnProperties()
         {
-            Assert.That(TestContext.CurrentContext.Test.Properties.Get("Answer"), Is.EqualTo(42));
+            Assert.That(TestContext.CurrentContext.Test.Properties.Get("Answer"), Is.EqualTo("42"));
+        }
+
+        [Test]
+        public void TestGettingPropertiesReturnsNewObjEveryTime()
+        {
+            PropertyBagAdapter adapter1 = CurrentContext.Test.Properties;
+            PropertyBagAdapter adapter2 = CurrentContext.Test.Properties;
+
+            Assert.AreNotEqual(adapter1, adapter2);
```

Comment:
If this changed, would we care?

Reply:
Probably not, and I'd probably argue for exactly that reason, this test should be removed.  Originally I had `CurrentContext.Test.Properties` return a new `PropertyBagAdapter` every time because, upon construction, it created a shallow-copy of the current state of the given `IPropertyBag`, instead of what it does now, holding a reference to the source `IPropertyBag`.  In that context, it made sense to ensure that calling `.Properties` always returned a new object, probably not anymore.

Reply:
It's a bit academic since `TestContext.CurrentContext` always returns a new object.